base/database/setup.go

```diff
@@ -20,6 +20,19 @@ func Configure() {
 	check(Db)
 }
 
+func Close() {
```

Comment:
Can you add a test for this ? Maybe running `pg_sleep` in a transaction and Closing the connection from a goroutine.

Reply:
ok, will try it